Examples/src/components/Examples/Charts2D/Filters/PercentageChange/index.tsx

```diff
@@ -241,7 +60,23 @@ export default function PercentageChange() {
                         Original Data
                     </ToggleButton>
                 </ToggleButtonGroup>
-                <div id={divElementId} className={localClasses.chartArea} />
+                {/* // Usage in SciChartReact */}
+                <SciChartReact
+                    key={chartKey} // Change the key to force re-render
```

Comment:
A note for myself @jim-risen.
We can refactor this to avoid the cahrt recreation.
Expose a functions to changed the related properties to demonstrate dynamic updates.